views/s3uploader.js

```diff
@@ -0,0 +1,25 @@
+'use strict';
+
+var app = angular.module('s3uploader', []);
+app.controller('fileSelection', function ($http) {
+  var fileName;
+  var file;
+  this.fileNameFetch = function () {
+    fileName = document.getElementById('myFile');
```

Comment:
Stick to angular if you are using that. This is not the angular way of uploading file.